app/views/groups/transferclients.html

```diff
@@ -21,7 +21,7 @@
                 <div class="col-sm-3 col-md-3">
                     <label class="control-label col-sm-12 center">{{ 'label.input.selectedclients' | translate }}</label>
                     <select multiple ng-model="selectedClients" class="form-control clear">
-                        <option ng-repeat="member in selectedMembers" value="{{member.id}}">{{member.accountNo}}{{member.displayName}}</option>
+                        <option ng-repeat="member in selectedMembers" value="{{member.id}}">{{member.accountNo}} | {{member.displayName}}</option>
```

Comment:
Why this change is required? 
If you want to add '|' between accountNo and displayName for selected members,
you should be doing same thing even for available clients.